wire-library/wire-grpc-client/build.gradle

```diff
@@ -35,6 +35,7 @@ kotlin {
     jvmMain {
       dependencies {
         api deps.okhttp
+        api project(':wire-runtime')
```

Comment:
Shouldn't be needed since it's already declared in `commonMain`.

Reply:
Fixed. IntelliJ is still grumpy but Gradle is happy.

Reply:
Hmmm, is that because reflection? Although I don't think we package `kotlin-reflect` with `wire-runtime`. Anyway, Kotlin reflection is tricky, some APIs are present in stdlib but aren't backed by an actual implementation, so they'll blow up at runtime unless `kotlin-reflect` is on the classpath. May not be the case here, but worth checking.